README.md

```diff
@@ -257,6 +257,111 @@ window.navigator.getUserMedia({
 });
 ```
 
+**List of available sources:**
+
+| Sources | Description | Browsers that supports it |
+| ------- | ----------- | --------- |
+| `"window"`  | Fetches the list of application windows | Chrome, Opera, Firefox, IE, Safari |
+| `"screen"`  | Fetches the list of display screens     | Chrome, Opera, Firefox, IE, Safari |
+| `"tab"`  | Fetches the list of browser tabs | Chrome, Opera |
+| `"browser"`  | Fetches the list of browser windows | Firefox (Requires to configure `about:config` to enable `media.getusermedia.browser.enabled`)  |
+| `"application"`  | Fetches the list of applications | Firefox |
+| `"camera"`  | Fetches the list of cameras | Firefox |
+| `["tab", "audio"]`  | Fetches `"tab"` with its audio | Chrome, Opera (Enable `constraints.audio` to retrieve tab audio) |
+| `["window", "screen"]`  | Fetches `"screen"` and `"window"` | Chrome, Opera, IE, Safari |
+| `["window", "screen", "tab"]` | Fetches `"screen"`, `"window"` and `"tab"` | Chrome, Opera |
+| `["window", "tab"]` | Fetches `"window"` and `"tab"` | Chrome, Opera |
+| `["screen", "tab"]` | Fetches `"screen"` and `"tab"` | Chrome, Opera |
+| `AdapterJS.WebRTCPlugin.plugin.screensharingKeys.screenOrWindow`  | Same as `["window","screen"]` | IE, Safari |
+| `AdapterJS.WebRTCPlugin.plugin.screensharingKeys.screensharingKey` | Deprecated. Same as `["window","screen"]`. | IE, Safari |
+| `AdapterJS.WebRTCPlugin.plugin.screensharingKeys.screen`  | Same as `"screen"` | IE, Safari |
+| `AdapterJS.WebRTCPlugin.plugin.screensharingKeys.window` | Deprecated. Same as `"window"`. | IE, Safari |
+
+
+## Configuring AdapterJS
+
+To configure your AdapterJS, configure this before referencing the script for `adapter.screenshare.js` or `adapter.debug.js` as an example:
+
+```
+var AdapterJS = {};
```

Comment:
I'm sure perhaps it might differ for plugin configuration but was thinking from an end-user pov, I think it might be easier to have something easier to understand and configure. Hence I left it that way and for configuring things.

```
<script>/* Modify */</script>
<script src="adapter.screenshare.js"></script>
```